examples/freestar.amp.html

```diff
@@ -0,0 +1,30 @@
+<!doctype html>
```

Comment:
can you move the ad tag to `ads.amp.html`, let me know if there is reason you think you need your own example file.